### DISCUSSION TITLE
Background Tasks on v3 works?

### DISCUSSION THREAD

**Discussion by TopazKnightDev** [ionic-team/capacitor#5449] [Q&A]
how can i keep alive my aplication on background? i used js timers like timeout and interval function but if i put 15min> the timer does'nt works:
Image example: Background Tasks Plugin v2
![Screenshot_15](https://user-images.githubusercontent.com/15913708/154844605-ada8af03-30fa-45fc-87ba-91310145aefd.png)

**Comment by angeloraso** [comment]
Hi. I made [this plugin](https://github.com/angeloraso/capacitor-plugin-background-mode) for that. Hope this can help you

**Reply by angeloraso** [reply]
It's an update from [this plugin](https://github.com/katzer/cordova-plugin-background-mode)
